js/plugin.iiif-iabookreader_strawberry.js

```diff
@@ -72,7 +87,7 @@ BookReader.prototype.loadManifest = function () {
       async: false,
       success: function (jsonLd) {
         self.jsonLd = jsonLd;
-        self.bookTitle = jsonLd.label;
+        self.bookTitle = self.getApiVersion() == "3.x" ? self.jsonLd.label[Object.keys(self.jsonLd.label)[0]].join("; ") : self.jsonLd.label;
```

Comment:
This requires probably sanity check... make sure actually a [0] exists there...

Reply:
Sanity-check added.